docs/howto/rally_benchmarking.md

```diff
@@ -279,3 +279,15 @@ esrally --target-hosts='{"defauelt":["%es_cluster_host:es_cluster_port%"]}' --tr
 
 Please refer to [esrally CLI reference](https://esrally.readthedocs.io/en/stable/command_line_reference.html) for more details.
 
+## Replaying an existing corpus
+If the `use-corpus-at-path` flag is used, the corpus will not be generated but rather loaded from the file passed as value of the flag.
+If the `rally-track-output-dir` flag is used alongside the `use-corpus-at-path` flag, the persisted rally track will reference a copy of the existing corpus saved in the directory passed as value of the `rally-track-output-dir` flag.
+
+## Loading a package from registry
+If the `package-from-registry` flag is used, the package installed in Kibana and its assets referenced in the track will be loaded from registry. The format of the flag value is `%packageName%-%packageVersion%`.
+You still need to run the command from the root of the local package in order to read the benchmark scenario (`benchmark` flag).
```

Comment:
Does this mean I need to run two commands? I tried the following:

```
elastic-package benchmark rally --benchmark pod-benchmark -v --package-from-registry=kubernetes-1.55.0
```

Can you share a working example that users can just copy/paste?

Reply:
your example is correct: I've reworded the docs, please let me know if it's clearer now